test/test_cpp_extensions_open_device_registration.py

```diff
@@ -152,6 +152,8 @@ def test_common_registration():
                 torch.utils.rename_privateuse1_backend('xxx')
             # register foo module, torch.foo
             torch._register_device_module('foo', DummyModule)
+            self.assertTrue(torch.utils.get_custom_mod_func("device_count")() == 1)
+            self.assertTrue(torch.utils.get_custom_mod_func("func_name") is None)
```

Comment:
Did we agree to make this private? (`_get_custom_mod_func)

Mostly because it's a convenience helper for backends, but isn't really intended for use by users.

Reply:
> Did we agree to make this private? (`_get_custom_mod_func)
> 
> Mostly because it's a convenience helper for backends, but isn't really intended for use by users.

yeah  it's okay,I will make it later. And anywhere else is ok? @bdhirsh

Reply:
> Did we agree to make this private? (`_get_custom_mod_func)
> 
> Mostly because it's a convenience helper for backends, but isn't really intended for use by users.

yeah I have make the func private, maybe you could have alook again. @bdhirsh

Reply:
just gave another round of review!